kibbeh/src/ui/SearchBar.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { SmSolidSearch } from "../icons";
+
+export interface SearchBarProps
+  extends React.ComponentPropsWithoutRef<"input"> {
+  inputClassName?: string;
+}
+
+export const SearchBar: React.FC<SearchBarProps> = ({
+  className,
+  inputClassName,
+  ...props
+}) => {
+  return (
+    <div
+      className={`relative text-primary-300 focus-within:text-primary-100 w-full ${className}`}
```

Comment:
`${className}` adds `undefined` to the class list. Might not be needed to add extra classes.

Reply:
I think passing through the className is useful when you want to add margin classes from the parent without having to add extra wrapper divs so I think we should leave it. I set the default values to be empty strings so that `undefined` doesn't get added to the classes list.